scipy/stats/_continuous_distns.py

```diff
@@ -1411,8 +1411,17 @@ def _stats(self, df):
         return mu, mu2, g1, g2
 
     def _entropy(self, df):
-        return (sc.gammaln(.5 * df) +
-                .5 * (df - np.log(2) - (df - 1) * sc.digamma(.5 * df)))
+
+        def regular_formula(df):
+            return (sc.gammaln(.5 * df)
+                    + 0.5 * (df - np.log(2) - (df - 1) * sc.digamma(0.5 * df)))
+
+        def asymptotic_formula(df):
+            return (0.5 + np.log(np.pi)/2 - 1/(6*df) - 1/(6*df**2)
+                    - 4/(45*df**3) + 1/(15*df**4))
```

Comment:
```suggestion
            return (0.5 + np.log(np.pi)/2 - (df**-1)/6 - (df**-2)/6
                    - 4/45*(df**-3) + (df**-4)/15)
```
This is probably better because we don't get a warning on underflow.